models/equflash/relaxation/optimizers/FIRE_torch.py

```
@@ -0,0 +1,275 @@
+        self.a = a * torch.ones(
+            batch_size,
+            dtype=torch.float64,
+            device=device,
+        )
+
+        self.downhill_check = downhill_check
+        self.position_reset_callback = position_reset_callback
+        if self.traj_dir and (not traj_dir or not len(traj_names)):
+            raise ValueError(
+                "Trajectory names should be specified to save trajectories"
+            )
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)